### PR TITLE
Added a hint about the required config v4 version hint.

### LINKED ISSUE #2751: `exec_always` doesn't seem to work without `# i3 config file (v4)`
Output of `i3 --moreversion 2>&- || i3 --version`:

Binary i3 version:  4.13 (2016-11-08) © 2009 Michael Stapelberg and contributors

URL to a logfile as per http://i3wm.org/docs/debugging.html:

https://github.com/i3/i3/files/956246/i3.txt, the relevant part only, ie: it not detecting it as a v4 script.

**What I did:**

Type my own config from scratch

**What I saw:**

`exec_always` doesn't seem to work without `# i3 config file (v4)`

**What I expected instead:**
The behavior being documented or [it being able to detect more v4 stuff](https://github.com/i3/i3/blob/next/src/config_parser.c#L637) (see additional note)
____

and now the useful bug report:

**description:**
`exec_always` (and possibility other function) doesn't seem to work without `# i3 config file (v4)` in the script

This isn't documented in the [userguide](https://i3wm.org/docs/userguide.html)\*, took me a long time to figure it out.

**reproducible test** (~/.config/i3/config)**:**
```
exec_always xterm
```

**test note:**
add `# i3 config file (v4)` anywhere and then it works
WARNING: the test case doesn't contain binding for exiting

**additional note:**
[The issue which added the feature](https://github.com/i3/i3/issues/435) seem to have been tag as 4.0, suggesting it is only added in v4

\* https://i3wm.org/docs/4.13/userguide.html for future viewer when it's available

### PR DESCRIPTION
**Pull request by Airblader**
fixes #2751

### CODE REVIEW

**Comment by Airblader** [comment]
rubber-stamp approval, @stapelberg

File: docs/userguide [hunk 1]
```diff
@@ -297,6 +297,13 @@ keyboard layout. To start the wizard, use the command +i3-config-wizard+.
 Please note that you must not have +~/.i3/config+, otherwise the wizard will
 exit.
 
+Please note that the config must contain the following line since version 4.0
```
Comment (stapelberg) [inline docs/userguide]: This is actually not entirely true. The “# i3 config file (v4)” marker is one way to make i3 interpret the config file as a version 4 file. Others can be seen at https://github.com/i3/i3/blob/fff3f79da9a87a1f790c6328f6615422f2b69b47/src/config_parser.c#L637-L640

Could you rephrase the paragraph to make it clear that i3 tries to detect v4 files automatically, but you can make sure it does the right thing by adding the hint?

**Comment by Airblader** [comment]
@stapelberg I've rephrased it now.
